src/components/UploadForms/UploadTab.js

```diff
@@ -18,14 +19,88 @@ const uploader = new FineUploaderTraditional({
             enabled: true
         },
         request: {
-            endpoint: 'http://localhost:3030/upload'
+            endpoint: BASE_URL + '/upload'
         },
         retry: {
             enableAuto: false
         }
     }
 });
 
+const ReviewPanel = ({ props }) => {

```

Comment:
As a technical debt task, this should be moved into ReviewUpload instead of living here.